docs/deployment/high_availability_guide.md

```diff
@@ -15,20 +15,70 @@
AQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhABACACEAEAIAIQAQAgAhAJ8gBABCACAEAEIAIAQAQgAgBABCACAEAEIAIAQAQgAgBABCACAEAEIAIAQAQgAgBABCACAEAEIAIAQAQgAgBABCACAEAEIAIAQAQgAgBABCACAE4BOEAEAIAIQAQAjAPwMBwO8nukMAkAf2xR/DUB+CAoBDCgCkAEAKwCdIAYAUAEgBgBQASAH4BCkAkAIAKQCQAvAJUgAgBQBSACAFAFIAIAUAUgAgBQBSACAFAFIAPkEKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACgCkAEAKAKQAQAoApABACsAnSAGAFABIAYAUAEgBgBQASAGAFIBPkAIAKQCQAgApAJACACkAkAIAKQCQAgApAB+ZAkASyCsKAMG+pgCwfzcE4P2bIN5QAMKH9OENCWA6VVo2gAHQJ2kbrksvAHv62gNv70/a09SqiH6dAHAuzuDHUXFu1+kIfoqBd/p7TxCAYBJ1WD93x0vZo1/poa8KHqPY1/gFHHkjahp5K+Z7f/16GZP/ORmTyEeTMfv9NIsoD69cXRf9JIvwfHj6KAa51zRp8FrUXwozvKbt/glIAT474PMD+bQlXl/sEoenja/Ku/XqOPymi3myzvAQh9/slhdiJ98R+/27Osy9Nr2Er67ivb54amFVpNMlfwabMMxrl0ahDzT5+ixN0dVB+PTDz5365lwY+xUNup/oUThvTnTTjuc7/+sKQ7+nMFQ+CYxvSu/8SnOoqivaJ2X4o7lpwzTCIChWXm/de98/fYrB/+uw6sKmvZ9vur7HUz7ufWv3U+/eiCqvVe+N3X5JTjmlhwM4x9TedFGefzsfULgSCO0mRpL/RIrvqPF3eJCn66Oftl+gXAQBmf7eVexv2+bbQTYozucwaJ+uffoO+abb+QN5wNDXeoP9nGbfDymiqAl/i6KR9J97JtAL5fe75R+W25fGht0v6oXfwJB3HAfzu/w1Sf25VH7zeFg/3ub3avFzP/75IPnPSRV7I9Xjx4o0fla09/jjtTr/w8EH+Ral9d8UO/XB5P42eflvyh39YApPv/Uzb4T+K6LtHxToBwmT2ddR8h/0F93wvSHy/Tx/V0T8Nk3904g4eLQQEA3Xse/9z3Sn039T+8i7n/7301O3IrdQOvJOaT48/vxUnIvmZpivDvkcbSMAh/e0y3820T9eX8H/YATz3BwGAs/7ZxI0/mWg/mlKO4CGTI4g0YrDFIFPX8xu39bFdOTz1l2C5E2G0zci+AxOTgIpkWAS5U+ORZ+PvSv0XzoN9vk0jx31vOfzjsd+et7xaIlg87MtTlufrRHsutsj2HGzSPAlett8tkrw1e1ib5J7auxmdN9zl8+7nm/ws2WSd9t8Ppr8LJS7jb6779lW37sE8Mzn/ev4RWOvruLJoEhgd89H0/iLNobP3+Mv2355+ldG/V5v/lAz5Nfu4pUPeNvMbfOVYF/veFKpb6sY/k0VAx4BaEB624jbZ3W47wDq8o4aPtre/YBHw562uNuh6M26Xx708uxBHXpt+LKN2+3cG3rvIm4HvLnEd83jc/r9ju4/jvE3A3kc5cGXrxNxsPMpFQc7PyfjYMejtTwl5E9NkuBaSPE7bOspQnhPtV9mgO84kjfW9+we3re+F40gDyhBvdCB4d32f6XdvTz8noj/rG7fvnztxO8neO3t67D6wIWaL0ovGCpJFPH9kdFzoPZLSi/MPdC6Rzboz8VLv7/4Qr9XZvh3Vfme2cq/o8r3QrUkCZ/+3lT//gE9Q+n7dNddzz5+kY/5exKi/9fTD1O49EotCPqvTz+g5BfTD1/myr872XqvKvzoSB7DmrsX+QvO6oc91fPXr5v+GKPf90xEfNvRfW8V4W7Dv8SH4feZ/CcFwz+8C/ubZlCbti6O4b2LDmHkdY8qfgzbIHlSjamhw+2UP+fA7rMMH8WBfTFrTqDoA4VjBEVjOI4wGIv/NWeGMdQDg+A0yRAkjVPMPeq5N0MiDwyOEDhCESSKUdTf6umov1BW+v4QLE6bW1b2YQP4n5ppRRCK+pGZ1mcb/hUuDCHRf1u4TyF/jw/7z3gkAsEfGIJip/iHolj2ddhNIH89viJY5gO7pF+YFT6WqfunrgG76CeL/c3O6j59/bf6qhe+CceffNNLH0Z9p6N6NtRfUpd4/u2/Jdai0HcUEPqpr/spknodTRMI+0CSKEIxJEPhOPuF//huL8W8Dsjwv3kRGvX1LPBfExu1Rfk7crtn9/JLcrtng/slgRH+Wms+flxE/tP+5oO4kS8XDxJ/dWr+yxNRxBfX8rsdB/ZOh/6g48C/4jhe16F+4kT3abuvlJd+iVvKw+jXBUEo/vMJG4Kw7A8kbL/ML91n7V4o5Yd3S9TbdXO/xS09V80fqFvn3CvnwJcj6DeL52BjFdbpdKtg9Pt2PerRe3zrhr8z+vpghXecZR9wnMQZlCBIYvrnizwR/Yk8EUUept+jFEmDPJR5nSbiJPvAkJOzRaZEkmWZL56C+t1O9u3ywh/2jV+px78JzX6m8v4bMsEv3dxfywy/Nzi7u4Ff4AQRknztBD9+cMa8p2b/Yi/4QZwWSRAPOEog97/XJQKKwB4Q9F7g/FGvRRL4A8HSOEOjKMKQX5QfSOr2qCeFEujksqZW/lavxXy93g691i/0Ws9m+1/zWp+e1k29OPzziil89n8=&lt;/diagram&gt;&lt;/mxfile&gt;&quot;}"></div>
```

Comment:
a little hack, dose our docs builder support import content from the text file?

Reply:
the body is foldable here, I will search a better solution later